framework/TApplicationConfiguration.php

```diff
@@ -234,6 +250,60 @@ protected function loadPathsXml($pathsNode, $configPath)
 			}
 		}
 	}
+	
+	/**
+	 * Reads the Composer static RegisteredLoaders for their Vendor Directory. Reads the Vendor
+	 * Directory composer file 'installed.json' (accumulated extensions composer.json) for the project.
+	 * The ['extra']['bootstrap'] field is read for each extension, if it's there.
+	 * @return array<string, string> the extension name and bootstrap class.
+	 * @since 4.2.0
+	 */
+	protected function getComposerExtensionBootStraps()
+	{
+		if ($cache = Prado::getApplication()->getCache()) {
+			$plugins = $cache->get(self::COMPOSER_INSTALLED_CACHE);
+			if ($plugins !== null) {
+				return $plugins;
+			}
+		}
+		$dependencies = new TChainedCacheDependency();
+		$listDeps = $dependencies->getDependencies();
+		$plugins = [];
+		foreach (ClassLoader::getRegisteredLoaders() as $vendorDir => $loader) {
+			echo($vendorDir . "\n");
+			$file = $vendorDir . DIRECTORY_SEPARATOR . 'composer' . DIRECTORY_SEPARATOR . 'installed.json';
+			$manifests = json_decode(file_get_contents($file), true, 512, JSON_THROW_ON_ERROR);
```

Comment:
JSON_THROW_ON_ERROR requires php 7.3
Since 7.2 is already EOL, we can just require php >= 7.3 for prado 4.2

Reply:
Perfect.

Reply:
This probably doesn't need to be JSON_THROW_ON_ERROR because the installed.json *should* always work

Reply:
Is this a worthy dependency? in your view?

Reply:
Keep it, people running EOL php versions already have worst problems